ntoskrnl/ps/query.c

```diff
@@ -49,6 +49,189 @@ PsReferenceProcessFilePointer(IN PEPROCESS Process,
     return Section ? STATUS_SUCCESS : STATUS_UNSUCCESSFUL;
 }
 
+/**
+ * @brief
+ * Dumps the name of a process information class.
+ *
+ * @param[in] ProcessInformationClass
+ * The enumeration code value of a process information class,
+ * of which the name of it is returned as a string.
+ *
+ * @return
+ * Returns a constant pointer to a string of the process
+ * information class name.
+ *
+ * @remarks
+ * If an unknown information class is not supported by
+ * PROCESSINFOCLASS, the function will return the value
+ * of the passed information class.
+ */
+static
+PCSTR
+PspDumpProcessInfoClassName(
+    _In_ PROCESSINFOCLASS ProcessInformationClass)
+{
+    static CHAR UnknownClassName[10];
```

Comment:
My bad, it's not 10, but 11: indeed the max ulong number in decimal is "4294967295", which is 10 chars + 1 terminating NUL long. (You could use `sizeof("4294967295")` then for the array size.)